<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="LB_Fichier1">
      <property name="text">
       <string>Contenu du fichier :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="TE_Fichier1"/>
    </item>
    <item>
     <widget class="QPushButton" name="PB_Fichier2">
      <property name="text">
       <string>Selectionner un autre fichier </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="LE_Fichier2"/>
    </item>
    <item>
     <widget class="QTextEdit" name="TE_Fichier2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
